package.json

```diff
@@ -45,5 +45,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "jest-environment-jsdom": "^29.3.1",
+    "ts-jest": "^29.0.5",
+    "ts-node": "^10.9.1"
```

Comment:
These are the new packages, or **dependencies**, that we've added to support testing our react components.